app/services/dashboard_runlist.rb

```diff
@@ -24,8 +35,19 @@ def to_json
 
   private
 
-  def submissions(run, page_id)
-    CRater::FeedbackSubmission.includes(:c_rater_feedback_items => :answer).where(run_id: run.id, interactive_page_id: page_id).order(:id).map do |submission|
+  def submissions(run, page_id, submissions_created_after)
+    CRater::FeedbackSubmission
+      .includes(
+        c_rater_feedback_items: {answer: [:run, :question]},
+        embeddable_feedback_items: {answer: [:run, :question]}
+      )
+      .where('created_at > ?', submissions_created_after)
```

Comment:
The index for this query currently uses runs & page.  I wonder if we should also add new index on all three columns `interactive_page_id`, `run_id`, and `created_at`   What do you think?